skimage/measure/block.py

```diff
@@ -59,7 +60,9 @@ def block_reduce(image, block_size, func=np.sum, cval=0, func_kwargs=None):
             [35]]])
     """
 
-    if len(block_size) != image.ndim:
+    if np.isscalar(block_size):
+        block_size = (block_size,) * image.ndim
+    elif len(block_size) != image.ndim:
         raise ValueError("`block_size` must have the same length "
```

Comment:
This error message needs an update to reflect that it can be either a scalar or have the same length ...

Reply:
I updated the error message from this to "block_size` must be a scalar or have the same length as `image.shape`".